### PR TITLE
[Validator] Documentation for PR #43611

### PR DESCRIPTION
**Pull request by AndreasA**
Documentation change for PR https://github.com/symfony/symfony/pull/43611

### CODE REVIEW

File: reference/constraints/UniqueEntity.rst [hunk 1]
```diff
@@ -361,4 +361,11 @@ blank, ``findBy()`` will be used. The method receives as its argument a
 fields configured in the ``fields`` option). The method should return a
 :phpfunction:`countable PHP variable <is_countable>`.
 
+This method should ensure two return a maximum of two entities,
```
Comment (iDara77) [inline reference/constraints/UniqueEntity.rst]: ... should ensure **to** return ...

File: reference/constraints/UniqueEntity.rst [hunk 2]
```diff
@@ -361,4 +361,11 @@ blank, ``findBy()`` will be used. The method receives as its argument a
...
+if a custom method is used. This is necessary to avoid potential memory and
+performance isuses. Retrieving two entities is enough to determine uniqueness:
```
Comment (iDara77) [inline reference/constraints/UniqueEntity.rst]: performance issues ...

File: reference/constraints/UniqueEntity.rst (hunk 1 above)
Comment (AndreasA) [reply reference/constraints/UniqueEntity.rst]: ups. definitely true 😄

File: reference/constraints/UniqueEntity.rst (hunk 2 above)
Comment (AndreasA) [reply reference/constraints/UniqueEntity.rst]: thanks

**Review by iDara77 — COMMENTED** [review]
just some typos

**Comment by AndreasA** [comment]
> just some typos
Thanks. I totally missed those, when creating the PR. though `two` is more a grammatic error that probably happened as I already thought about the second part of that sentence 😄

**Comment by javiereguiluz** [comment]
Andreaas, sorry it took us so long to review this. I was checking the source code of the change and I see that it's a nice performance improvement. However, I don't think we need to explicitly mention the details of this change in the docs. Your contribution is great and makes Symfony faster and more performant, so that's enough 🙏  Thanks!

**Pull request closed without merging**